postgresmodels/attackchains.go

```diff
@@ -0,0 +1,68 @@
+package postgresmodels
+
+import (
+	"gorm.io/gorm"
+)
+
+type AttackChainNode struct {
```

Comment:
Please test it with AutoMigrate.
Example: init.go in github.com:kubescape/event-ingester-service.git

Reply:
This will give us the sense more accurate

Reply:
@Bezbran 
tested, it is good.